packages/runner-ct/src/app/app.tsx

```diff
@@ -72,7 +76,19 @@ const App: React.FC<AppProps> = observer(
             defaultSize={(windowSize.width - defaultIframeWidth) / 100 * 70}
             minSize={200}
           >
-            <SpecsList state={state} config={config} />
+            <div>
+              <SearchSpec
+                value={props.state.specSearchText}
```

Comment:
Maybe it will be easier to pull the state down, and connect to mobx?

Reply:
Do you think we should have the mobx state in the "dumb" or "presentation" components? I am not super familiar with mobx yet but it feels a little weird to pass this giant observable state to all the components - is this normally how it works?

It also feels a bit weird that such a simple component (basically just rendering some `<li>` elements) needs to have a huge, complex state containing things like EventManager, global config, etc.